woocommerce/woocommerce
# Fix e2e docker:ssh command to work in any repo

### All Submissions:


### Changes proposed in this Pull Request:

Dependent on #27113 being merged as this uses features added to that PR. 

The current `docker:ssh` script in the `e2e-environment` only works in WC core. The `e2e-environment` names containers using the `basename()` of the repository root (eg WC Admin would be `woocommerce-admin_wordpress-www`). The current script is hardcoded to `woocommerce_wordpress-www`.

Closes # .

### How to test the changes in this Pull Request:

1. `npm run docker:up`
2. ` npm explore @woocommerce/e2e-environment -- npm run docker:ssh`
3. `cat wp-config.php`
4. You should see the WP config file for the test environment.

### Other information:




### Changelog entry

> Fix e2e docker:ssh command to work in any repo.
